src/js/main.js

```diff
@@ -298,6 +301,16 @@ _.extend(Smooch.prototype, Backbone.Events, {
         }.bind(this));
     },
 
+    _subscribeToEvents: function() {
+        this._chatController.on('message:received', _(function(message) {
```

Comment:
use `this.listenTo(this._chatController, 'message:received', ...)` instead. That way, if the chatController is destroy and reinitialize again, the old instance can be garbage collected and vice-versa, if the Smooch object is destroyed..